dayPlanner.html

```diff
@@ -82,7 +82,7 @@
           <div id="settingBox">
             <div class="settingBoxContent">
               <span id="close">&times;</span>
-              <input id="todoArea">
+              <input type="search" id="todoArea">

```

Comment:
- input kısmında yazılı olanları tek butonla temizleyebileceğimiz bir çarpı işareti için bunu ekledim.